src/routes/paymentsCheckout.js

```diff
@@ -0,0 +1,40 @@
+const express = require("express");
+const mercadopago = require("mercadopago");
+const router = express.Router();
+require("dotenv").config();
+
+
+router.post("/payment_mercadopago", (req, res) => {
+    mercadopago.configurations.setAccessToken(process.env.ACCESS_TOKEN_MP);

```

Comment:
env variables can be declared in the config.js file, then you will need to import the config module to this file